src/config/helmet.ts

```diff
@@ -5,18 +5,20 @@ import * as config from '../config';
 
 export const configureHelmet = (app: express.Application) => {
     app.use(helmet({
+        referrerPolicy: {
+            policy: 'strict-origin-when-cross-origin'
+        },
         contentSecurityPolicy: {
             useDefaults: true,
             directives: {
                 defaultSrc: ["'self'"],
-                fontSrc: ["'self'"],
+                fontSrc: ["'self'", config.CDN_HOST],
                 styleSrc: ["'self'", config.CDN_HOST],
                 scriptSrc: [
                     "'self'",
                     "'sha256-l1eTVSK8DTnK8+yloud7wZUqFrI0atVo6VlC6PJvYaQ='",
                     "'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU='",
                     config.CDN_HOST
```

Comment:
Thanks @DanielMurray97, could you please add `sha` comments for future reference. I have created the following story on the backlog https://technologyprogramme.atlassian.net/browse/NTRNL-503

```
...
    scriptSrc: [
        "'self'",
        // Hash of the inline scripts respectively for:
        // `<script> document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>`
        "'sha256-l1eTVSK8DTnK8+yloud7wZUqFrI0atVo6VlC6PJvYaQ='",
        // `<script>window.GOVUKFrontend.initAll()</script>` 
        "'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU='",
        config.CDN_HOST
    ],
...
```